src/configure

```diff
@@ -194,7 +194,7 @@ function check_compiler {
              You need g++ >= 4.7, Apple clang >= 5.0 or LLVM clang >= 3.3."
   else
     COMPILER_VER_INFO=$($COMPILER --version 2>/dev/null)
-    if [[ $COMPILER_VER_INFO == *"g++"* ]]; then
+    if [[ $COMPILER_VER_INFO == *[gc]"++"* ]]; then
```

Comment:
the name of cpp compiler like that from conda-build: "x86\_64-conda\_cos6-linux-gnu-c++", ends with "c++"